charts/opendj/templates/cronjob.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Chart.Name }}-statefulset-restart
+  labels: {{ include "opendj.labels" . | nindent 4 }}
+spec:
+  # If it's time for a new job run and the previous job run hasn't finished yet, the cron job skips the new job run
+  concurrencyPolicy: Forbid
+  schedule: '0 02 * * *' # Run once a day at 2 am UTC / 10 pm EST
```

Comment:
No worries if you've already discussed this with chelsea, but does it need to be restarted everyday? The problem only occurs once per month.

Reply:
That's what the [ticket](https://broadworkbench.atlassian.net/browse/DDO-1585?focusedCommentId=51775&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-51775) says as a proposed solution.  
>  A proposed solution is to add a CronJob that will restart the deployment nightly.

Reply:
And in this [comment](https://broadworkbench.atlassian.net/browse/DDO-1585?focusedCommentId=51775). I would mention that there was no formal discussion. However, you do bring up a good point.